project/build.properties

```diff
@@ -1 +1 @@
-sbt.version=0.13.12
+sbt.version=0.13.13
```

Comment:
This is also that we could enforce in org-policies to ensure everyone unless otherwise specified remains in the same version of sbt.